### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by moniquep** [wonderbit/sketch-select-similar-layers#10]
at all

**Comment by Indyandie** [comment]
Yeah not working for either :(

**Comment by wonderbit** [comment]
Hi, ummm… can you give me more details? I've been testing it on Sketch 42 and everything works as expected. Thanks!

**Comment by moniquep** [comment]
I have a file where certain colours don't seem to work. I tried replacing all instances of the colour #ECCB80 with #FFAD94 and it didn't work, but when I tried replacing it with a generic colour like #FFF000 it worked.
[redacted]
You have to go to the symbols page to see it

**Comment by wonderbit** [comment]
Hi, thanks for the file. I can confirm that it doesn’t work in this particular case. I will investigate.

**Comment by wonderbit** [comment]
Hi again, I found the problem. Turns out, that when you create a layer with a particular color, and then change the layer fill to gradient or something else, the layer still has that color property and therefore the plugin was returning false positives.

**TLDR;** I updated the plugin to fix this and also to work properly with gradient, image and noise fills.
